modules/govuk/manifests/apps/finder_frontend.pp

```diff
@@ -62,6 +82,21 @@
     "${title}-SECRET_KEY_BASE":
         varname => 'SECRET_KEY_BASE',
         value   => $secret_key_base;
+    "${title}-ACCOUNT-OAUTH-CLIENT-ID":
+        varname => 'GOVUK_ACCOUNT_OAUTH_CLIENT_ID',
+        value   => $account_oauth_client_id;
+    "${title}-ACCOUNT-OAUTH-CLIENT-SECRET":
+        varname => 'GOVUK_ACCOUNT_OAUTH_CLIENT_SECRET',
+        value   => $account_oauth_client_secret;
+    "${title}-ACCOUNT-JWT-KEY-UUID":
+        varname => 'GOVUK_ACCOUNT_JWT_KEY_UUID',
+        value   => $account_jwt_key_uuid;
+    "${title}-ACCOUNT-JWT-KEY-PEM":
+        varname => 'GOVUK_ACCOUNT_JWT_KEY_PEM',
+        value   => $account_jwt_key_pem;
+    "${title}-FEATURE-FLAG-ACCOUNTS":
+        varname => 'FEATURE_FLAG_ACCOUNTS',
+        value   => $feature_flag_accounts;
```

Comment:
We'll also need an appropriate Plek environment variable in here: https://github.com/alphagov/finder-frontend/blob/3535ad228a10ea1a269cadac0933103b5aee2a22/app/lib/services.rb#L35

Reply:
Ah, yes, forgot about this one.  Have added it now.